.github/workflows/update-usaco.yml

```diff
@@ -0,0 +1,49 @@
+name: Fetch from usaco-problems
+on: workflow_dispatch
+
+jobs:
+  update_problems:
+    name: Update problems
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout usaco-guide
+        uses: actions/checkout@v2
+      - name: Checkout usaco-problems
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ github.repository_owner }}/usaco-problems
+          path: usaco-problems
+      - name: Generate a token
+        id: generate_token
+        uses: actions/create-github-app-token@v1
+        with:
+          app-id: ${{ secrets.APP_ID }}
+          private-key: ${{ secrets.APP_PRIVATE_KEY }}
+          owner: ${{ github.repository_owner }}
+      - name: Build list of problem ids
+        run: |
+          git branch auto
+          git checkout auto
+          yarn
+          yarn develop &
+          sleep 20
+          pkill -f 'yarn develop'
+          ls
```

Comment:
maybe we should add a comment here? it's not immediately obvious to me why `yarn develop` is needed to do this...

(Also, `yarn build` instead of `yarn develop` might be more reliable, but if it works as is then I guess it's fine...)

Reply:
yeah I think we should probably switch to yarn build rather than yarn develop (since killing the process after 20 secs is a bit sketchy); I just used develop so my workflows would run faster so I could test them quicker 😅